cmd/openshift-install/create.go

```diff
@@ -73,15 +73,26 @@ var (
 		assets: []asset.WritableAsset{&templates.Templates{}},
 	}
 
-	ignitionConfigsTarget = target{
-		name: "Ignition Configs",
+	nodeConfigTarget = target{
+		name: "Node Config",
 		command: &cobra.Command{
-			Use:   "ignition-configs",
-			Short: "Generates the Ignition Config asset",
+			Use:   "node-config",
+			Short: "Generates the ignition configs for the master and worker nodes",
 			// FIXME: add longer descriptions for our commands with examples for better UX.
 			// Long:  "",
 		},
-		assets: []asset.WritableAsset{&bootstrap.Bootstrap{}, &machine.Master{}, &machine.Worker{}, &kubeconfig.Admin{}, &cluster.Metadata{}},
+		assets: []asset.WritableAsset{&machine.Master{}, &machine.Worker{}},
+	}
+
+	preClusterTarget = target{
+		name: "Pre-Cluster",
+		command: &cobra.Command{
+			Use:   "pre-cluster",
+			Short: "Generates the final assets prior to the cluster",
+			// FIXME: add longer descriptions for our commands with examples for better UX.
+			// Long:  "",
```

Comment:
As we get closer to a production release, it's probably time to start filling these in, at least for new commands.  Maybe echo `Short` and then add some wording about the asset filenames that will be generated (for `manifests` we can probably use wildcards instead of giving a full list)?

Reply:
I added a long description for `pre-cluster`. The short description for `node-config` covers everything, so I have removed the comment about needing to fill out the long description for that sub-command.